miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultPoolService.java

```diff
@@ -311,6 +315,21 @@ public void validateIndices(Pool pool, Pool beforeChange, Collection<ValidationE
     }
   }
 
+  @Override
+  public boolean checkMismatchedWithOrders(Pool pool, List<PoolOrder> poolOrders) throws IOException {
+    Set<Pair<LibraryAliquot, Integer>> poolAliquots = new HashSet<>(), poolOrderAliquots = new HashSet<>();
```

Comment:
I'm not fond of this style of multiple variable declaration on a single line as I find it requires more cognitive overhead to parse, especially since the variables have such similar names. Do you have strong objections to doing the more verbose one-variable-per-line declaration?

Reply:
Changed to two separate declarations